kubernetes/troubleshooting/kyverno.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: enforce-cmda-node-affinity
+spec:
+  admission: true
+  background: true
+  emitWarning: false
+  rules:
+  - match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+          namespaces:
+          - cmda
+    exclude:
+      any:
+      - resources:
+          names:
+          - "*agent*"
+    mutate:
+      patchStrategicMerge:
+        spec:
+          nodeSelector:
+            kubernetes.io/hostname: k3s-muddy-voice-95568538-d118-e06cc6-node-pool-77e9-xynqv
```

Comment:
Worth putting in a placeholder value so it's easier to find what needs updating?